### DISCUSSION TITLE
Parcel 2 library with React MaterialUI, can't get it to work as package

### DISCUSSION THREAD

**Discussion by JanMisker** [parcel-bundler/parcel#7505] [Q&A]
I am trying to build a Parcel 2 library of React components utilising Material UI (@mui/material). Does anyone have an example project that works, with the current version 5 of Material UI ? I think everything works with regards to building the library, in a minimal example I get a small file with just my own (transpiled) code.

When I then import my library in a 'client' app, the trouble starts.

For me it works when I symlink (`npm instal ../library`) the library package, if I also link react to the one in ../library/node_modules/react (to avoid duplicate react instances)
But I think that is because Parcel then treats it as 'source' in the sense that it is just compiled from source as if the files where in the 'client' project. When publishing (through private npm repository) as an npm package it starts failing.
It seems my library is not exported, I just get undefined when I import or require it. 
When debugging I do see calls being made to a `$parcel$export` function that I assume should expose the modules to the code that is calling the import, but as said there I just get undefined.

I realise this description is probably not easy to follow, that's why I write in Discussions rather than Issues.
I am trying to make a minimal example project, but I was hoping someone knows of a project that works?

**Comment by leon-up9** [comment]
im working with @material-ui/core

i get an error: Failed to compile.
Module not found: Can't resolve '@material-ui/core/Backdrop'

Does anyone have a solution?
